### PR TITLE
DOC: change PCI uart description from mmio to bdf

### PR DESCRIPTION
**Pull request by mcao6**
HV code changed to configure PCI uart mmio@ to bdf@,
so change its description to match it.

Tracked-On: #2031
Signed-off-by: Minggui Cao <minggui.cao@intel.com>

### CODE REVIEW

**Comment by HuiHuangShi** [comment]
2113 <br>The PR does not touch any hypervisor source file. Skip coding guideline checks.<br>

File: doc/getting-started/apl-nuc.rst [hunk 1]
```diff
@@ -177,7 +177,8 @@ partition. Follow these steps:
       whether it should be disabled. There are three forms for this parameter:
 
       #. ``uart=disabled``: this disables the serial port completely
-      #. ``uart=mmio@<MMIO address>``: this sets the serial port MMIO address
+      #. ``uart=bdf@<BDF value>``:  this sets the PCI serial port based on its BDF.
+      For example, use ``bdf@0:18.2`` for a BDF of 0:18.2 ttyS2.
```
Comment (gvancuts) [inline doc/getting-started/apl-nuc.rst]: You need this line to be aligned with the content of the bullet list, i.e. align its start with the first quote from the line above.

File: doc/getting-started/up2.rst [hunk 2]
```diff
@@ -75,21 +75,22 @@ You will need to keep these in mind in a few places:
...
+You can check this from the ``lspci`` output from the initial Clearlinux installation.
 
 .. code-block:: none
 
-   # dmesg | grep dw-apb-uart
-   [2.150689] dw-apb-uart.8: ttyS1 at MMIO 0x91420000 (irq = 4, base_baud = 115200) is a 16550A
-   [2.152072] dw-apb-uart.9: ttyS2 at MMIO 0x9141e000 (irq = 5, base_baud = 115200) is a 16550A
+   #lspci | grep UART
+00:18.0 . Series HSUART Controller #1 (rev 0b)
+00:18.1 . Series HSUART Controller #2 (rev 0b)
```
Comment (gvancuts) [inline doc/getting-started/up2.rst]: These two lines need to start at the same indentation as the `#lspci...`

File: doc/tutorials/using_partition_mode_on_up2.rst [hunk 3]
```diff
@@ -106,11 +106,10 @@ Enable partition mode in ACRN hypervisor
...
 
-     $ dmesg | grep ttyS
-     [   10.552369] dw-apb-uart.8: ttyS4 at MMIO 0x91524000 (irq = 4, base_baud = 115200) is a 16550A
-     [   10.552568] serial serial0: tty port ttyS4 registered
-     [   10.572344] dw-apb-uart.9: ttyS5 at MMIO 0x91522000 (irq = 5, base_baud = 115200) is a 16550A
-     [   10.572545] serial serial1: tty port ttyS5 registered
+     $ sudo lspci | grep UART
+      00:18.0 . Series HSUART Controller #1 (rev 0b)
+      00:18.1 . Series HSUART Controller #2 (rev 0b)
+   The second with ``00:18.1`` is the one on the 40-pin expansion connector.
```
Comment (gvancuts) [inline doc/tutorials/using_partition_mode_on_up2.rst]: You need an extra blank line and to start this sentence without any indentation.

File: doc/getting-started/apl-nuc.rst (hunk 1 above)
Comment (mcao6) [reply doc/getting-started/apl-nuc.rst]: OK, thanks! is it like this one? 
#. ``uart=bdf@<BDF value>``:  this sets the PCI serial port based on its BDF.  
      For example, use ``bdf@0:18.2`` for a BDF of 0:18.2 ttyS2.

File: doc/getting-started/up2.rst (hunk 2 above)
Comment (mcao6) [reply doc/getting-started/up2.rst]: Got. thanks!

**Comment by mcao6** [comment]
Thanks to Geoffroy. Update again.

File: doc/getting-started/apl-nuc.rst [hunk 4]
```diff
@@ -177,7 +177,8 @@ partition. Follow these steps:
       whether it should be disabled. There are three forms for this parameter:
 
       #. ``uart=disabled``: this disables the serial port completely
-      #. ``uart=mmio@<MMIO address>``: this sets the serial port MMIO address
+      #. ``uart=bdf@<BDF value>``:  this sets the PCI serial port based on its BDF.
+           For example, use ``bdf@0:18.2`` for a BDF of 0:18.2 ttyS2.
```
Comment (gvancuts) [inline doc/getting-started/apl-nuc.rst]: Just a couple too many spaces in the indentation, the beginning of the line should align with the first tick character

File: doc/tutorials/using_partition_mode_on_up2.rst (hunk 3 above)
Comment (dbkinder) [reply doc/tutorials/using_partition_mode_on_up2.rst]: No.  What's written renders correctly.  This sentence is outside of the code block, but still within the numbered list body, so still needs to be indented (as it is).

**Comment by mcao6** [comment]
will check and update soon.

**Comment by acrnsi** [comment]
Ok to verify

**Pull request merged**